sig-cluster-lifecycle/charter.md

```diff
@@ -0,0 +1,65 @@
+- Core add-ons that are required for cluster bootstrapping.
+
+#### Cross-cutting and Externally Facing Processes
+
+- This SIG works closely with sig-release during the end of the cycle, because of how the release is currently managed. The process requires a lock-step coordination which is outlined [here](https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md)
+- The SIG endorses third party cluster add-ons for networking, network policy, service discovery, etc. The SIG maintains the health check of container images for some add-ons that are required for cluster bootstrapping. While the SIG could provide support to users that have add-on related issues, the SIG can decide to delegate issues to the add-on maintainers or other SIGs.
+- The SIG owns the framework for upgrade / downgrade testing of Kubernetes. However, the SIG does not own the individual tests for each feature. While the SIG is often involved in triaging and debugging upgrade or downgrade test failures, the SIG will normal delegate issues to the feature owner for the failing test.
+- The SIG collaborates regularly with SIG-Auth in an effort to follow best practices in order to promote secure default deployments.
+- The SIG co-owns cloud provider specific code related to cluster and machine provisioning with the respective SIGs for each cloud provider but does not own the cloud controller manager or any other provider specific code
+
+### Out of scope
+
+- Networking related issues (see [sig-network](../sig-network))
+- UI / UX issues other than cluster bootstrapping or management (see [sig-ui](../sig-ui) and  [sig-cli](../sig-cli))
+- Kubernetes control plane issues:
+   - Node related issues (see [sig-node](../sig-node))
+   - Control plane component related issues (see [sig-apimachinery](../sig-apimachinery))
+
+## Roles and Organization Management
+
+This sig follows adheres to the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance].
+
+### Additional responsibilities of Chairs
+
+- Selecting feature for a kubernetes milestone.
+- Host the weekly SIG meeting, ensure that recordings are uploaded in a timely fashion
+- Make sure the breakout sessions the SIG hosts during the week have chairs.
+- Organizing SIG sessions at KubeCon events (intro / deep dive sessions)
+- To work out roadmaps for a given year or release, or creating/owning technical implementation plans (e.g. KEPs) in coordination with both SIG Cluster Lifecycle contributors and other SIGs.
+
+### Deviations from [sig-governance]
+
+- As SIG cluster lifecycle contains a number of subprojects, the SIG has empowered subproject leads with a number of additional responsibilities, including but not limited to:
+   * Releases: The subproject owner is responsible for determining the subproject release cadence, producing releases, and communicating releases with sig-release and sig-cluster-lifecycle
```

Comment:
`sig-release and sig-cluster-lifecycle` -> `SIG Release and SIG Cluster Lifecycle`